src/solids4FoamModels/solidModels/vertexCentredNonLinGeomTotalLagPressureDisplacementSolid/vertexCentredNonLinGeomTotalLagPressureDisplacementSolid.C

```diff
@@ -1531,11 +1531,25 @@ bool vertexCentredNonLinGeomTotalLagPressureDisplacementSolid::evolve()
             )
         );
 
-        //Calculate pBarSensitivity
+        // Calculate pBarSensitivity
         tensorField pBarSensitivity = -0.5*K*(
```

Comment:
A good sanity check for `pBarSensitivity` is to check whether it reduces to the expected small-strain version when the deformation is small. As I said above, I think it should reduce to `K` (or `K*I`), but please check the small strain version (i.e., `p = -K*tr(pointGradD)`).